dev/io.openliberty.microprofile.openapi.4.0.internal.services/src/io/openliberty/microprofile/openapi40/internal/services/impl/package-info.java

```diff
@@ -1,13 +1,15 @@
 /*******************************************************************************
- * Copyright (c) 2024, 2025 IBM Corporation and others.
+ * Copyright (c) 2025 IBM Corporation and others.
```

Comment:
```suggestion
 * Copyright (c) 2024, 2025 IBM Corporation and others.
```

Reply:
The original file was refactored into the other bundle and this package created fresh (albeit with the same name) which is why the copyright date was not retained.

I guess the name is still the same so you could argue it should stay? Unsure.